src/common/runtime/standalone.ts

```diff
@@ -83,7 +112,23 @@ function makeCaseHTML(t: TestTreeLeaf): VisualizedSubtree {
       await t.run(rec);
     }
 
+    result.total++;
+    result.timems = caseResult.timems;
+    switch (caseResult.status) {
+      case 'pass':
+        result.pass++;
+        break;
+      case 'fail':
+        result.fail++;
+        break;
+      case 'skip':
+        result.fail++;
```

Comment:
"skip" is a passing status. (we use it for cases where a webgpu feature isn't available, or the test is running on a worker and `document` isn't available, etc.)

Reply:
That was a typo, thanks for catching it! I'm recording the skip count now, but it's not factoring into the subtree's visuals in any way.